skfda/exploratory/visualization/fpca.py

```diff
@@ -0,0 +1,67 @@
+from matplotlib import pyplot as plt
+from skfda.representation import FDataGrid, FDataBasis, FData
+
+
+def plot_fpca_perturbation_graphs(mean, components, multiple,
```

Comment:
Try to keep the same API as other visualization functions. Use `_get_figure_and_axes` to parse arguments.

Reply:
Done